packages/debian/entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+rpm2deb() {
+	set -x
+	chown build:build /home/build
+	pushd /debbuild/PKG 1>/dev/null
+	for RPM in /rpmbuild/SRC/calico*.rpm; do
+		alien --bump=0 "$RPM"
+	done
+	cp -p /rpmbuild/SRC/*.deb /debbuild/PKG
+	chown "${TARGET_UID}:${TARGET_GID}" \
+		/debbuild/DEBS/*
+}
+
+builddeb() {
+	#need to be done with user "build"
+	set -x
+	chown build:build /home/build
+        pushd /debbuild/PKG 1>/dev/null
```

Comment:
Why moving in this directory since the line just after you use a full path and right after you move again ?
For me, this line is useless, must be a leftover of a copy/past from your function above.